docs/geedocs/5.3.3/answer/releaseNotes/relNotesGEE5_3_3.html

```diff
@@ -139,6 +139,11 @@ <h1>Release notes: Open GEE 5.3.3<a class="headerlink" href="#release-notes-open
 resulting in lower memory utilization and significantly improved
 performance.</td>
 </tr>
+<tr class="row-even"><td>1577</td>
+<td>Use defusedxml in server code</td>
+<td>Added defusedxml as a dependecy.  Using defusedxml in place of
```

Comment:
what was the benefit from using this xml library instead of the previous one. I would mention it here since this is a user facing document.

Reply:
This was a security issue